tfhe-csprng/benches/benchmark.rs

```diff
@@ -1,15 +1,15 @@
 use criterion::{black_box, criterion_group, criterion_main, Criterion};
 use tfhe_csprng::generators::{
-    AesniRandomGenerator, BytesPerChild, ChildrenCount, RandomGenerator,
+    BytesPerChild, ChildrenCount, DefaultRandomGenerator, RandomGenerator,
 };
 use tfhe_csprng::seeders::{RdseedSeeder, Seeder};
 
 // The number of bytes to generate during one benchmark iteration.
 const N_GEN: usize = 1_000_000;
 
 fn parent_generate(c: &mut Criterion) {
-    let mut seeder = RdseedSeeder;
-    let mut generator = AesniRandomGenerator::new(seeder.seed());
+    let mut seeder = RdseedSeeder::new();
+    let mut generator = DefaultRandomGenerator::new(seeder.seed());
```

Comment:
this still means we don't have control over which implementation is tested, sorry if I was not clear earlier

Reply:
we have with the feature software-prng, or maybe I don't understand what kind of control you mean ?

Reply:
ok should be fine in the end, I was unclear on the mechanism of the software prng feature